core/src/test/scala/za/co/absa/spline/harvester/dispatcher/ConsoleLineageDispatcherSpec.scala

```diff
@@ -29,20 +31,29 @@ class ConsoleLineageDispatcherSpec
 
   behavior of "ConsoleLineageDispatcher"
 
-  it should "send lineage to the stdout" in {
-    assertingStdOut(include("""["event",{"planId":null,"timestamp":999}]""")) {
-      new ConsoleLineageDispatcher(new BaseConfiguration {
-        addProperty("stream", "OUT")
-      }).send(ExecutionEvent(null, 999, None, None))
+  private val uuid1 = UUID.randomUUID()
+
+  Seq(
```

Comment:
Using loops, branching or other conditional logic is generally not recommended practice in unit-tests.
DRY principle works great in test code - the simpler (flatter) the better.
In you really need to share a test code (rather rarely), there are better ways for test code sharing (e.g. property-style testing, or behavior-style testing).
In this case I'd say there is nothing to reuse, so simply adding additional "should" statements would suffice.

Reply:
Sure, your castle (project), your rules. I have removed the branching, now there are only flat should-statements.

(But still, I had the feeling that this actually fulfilled the DRY principle as there was less repeating. I see, however, the probable reasoning behind keeping the tests rather simple - both for readability and not to actually recreate the logic in `main`).

Reply:
Did I say DRY? That was a typo, I apologize. I actually meant DAMP :)